stack.yaml

```diff
@@ -17,7 +17,7 @@
 #
 # resolver: ./custom-snapshot.yaml
 # resolver: https://example.com/snapshots/2018-01-01.yaml
-resolver: lts-20.16
+resolver: lts-22.9
```

Comment:
There is also stack.linux-static.yml that must be updated.

Did you make sure all the build jobs produce usable clients?